frontends/infinite-corridor/src/pages/LearningResourceDrawer.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback } from "react"
+import { useResource } from "../api/learning-resources"
+import RoutedDrawer from "../components/RoutedDrawer"
+import {
+  ExpandedLearningResourceDisplay,
+  ExpandedLearningResourceDisplayProps as LRDisplayProps,
+  LearningResource
+} from "ol-search-ui"
+import { useSearchParams } from "ol-util"
+
+const RESOURCE_ID_PARAM = "resource_id"
+const RESOURCE_TYPE_PARAM = "resource_type"
+const RESOURCE_PARAMS = [RESOURCE_ID_PARAM, RESOURCE_TYPE_PARAM] as const
+
+const formatShareLink: LRDisplayProps["formatShareLink"] = resource => {
+  const search = new URLSearchParams()
+  search.set(RESOURCE_ID_PARAM, String(resource.id))
+  search.set(RESOURCE_TYPE_PARAM, resource.object_type)
+  return `${window.location.origin}${window.location.pathname}?${search}`
+}
+
+const drawerImg = {
+  ocwBaseUrl: SETTINGS.ocw_next_base_url,
+  embedlyKey: SETTINGS.embedlyKey,
+  width:      440,
+  height:     239
+}
+
+export type ResourceIdentifiers = {
+  type: string
+  id: number
+  runId?: number | string | undefined
+}
+
+type DrawerContentProps = {
+  resourceType: string
+  resourceId: number
+}
+const DrawerContent: React.FC<DrawerContentProps> = ({
+  resourceId,
+  resourceType
+}) => {
+  const resourceQuery = useResource(resourceType, resourceId)
+  const resource = resourceQuery.data
+  return (
+    <section className="ic-lr-drawer" aria-label="Detailed description">
+      {resource && (
+        <ExpandedLearningResourceDisplay
+          imgConfig={drawerImg}
+          resource={resource}
+          formatShareLink={formatShareLink}
+        />
+      )}
+    </section>
+  )
+}
+
+const LearningResourceDrawer: React.FC = () => {
```

Comment:
@abeglova We had a [slack conversation](https://mitodl.slack.com/archives/D035M0FT7RB/p1661530534282309) about this drawer once:

> I was initially imagining:
> - drawer component in infinite-corridor
> - "LearningResourceDetails" or something in ol-search-ui ... that's the drawer content
> - LearningResourceCards have an onClick prop that we use to set the drawer parameters
> - drawer makes API call and passes relevant info the the details

When I did the work to hook up the drawer to the router, I also implemented that plan.